homeassistant/components/deconz/device_trigger.py

```diff
@@ -300,6 +300,78 @@
+    (CONF_LONG_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1005},
+    (CONF_LONG_PRESS, CONF_TURN_ON): {CONF_EVENT: 2001},
+    (CONF_SHORT_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2002},
+    (CONF_LONG_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2005},
+    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3001},
+    (CONF_SHORT_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3002},
+    (CONF_LONG_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3003},
+    (CONF_DOUBLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3004},
+    (CONF_TRIPLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3005},
+    (CONF_LONG_PRESS, CONF_DIM_UP): {CONF_EVENT: 4001},
+    (CONF_SHORT_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4002},
+    (CONF_LONG_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4003},
+    (CONF_DOUBLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4004},
+    (CONF_TRIPLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4005},
+}
+
+AQARA_OPPLE_6_BUTTONS_MODEL = "lumi.remote.b686opcn01"
+AQARA_OPPLE_6_BUTTONS = {
+    (CONF_LONG_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1001},
+    (CONF_SHORT_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1002},
+    (CONF_LONG_RELEASE, CONF_TURN_OFF): {CONF_EVENT: 1003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_OFF): {CONF_EVENT: 1005},
+    (CONF_LONG_PRESS, CONF_TURN_ON): {CONF_EVENT: 2001},
+    (CONF_SHORT_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2002},
+    (CONF_LONG_RELEASE, CONF_TURN_ON): {CONF_EVENT: 2003},
+    (CONF_DOUBLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2004},
+    (CONF_TRIPLE_PRESS, CONF_TURN_ON): {CONF_EVENT: 2005},
+    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3001},
+    (CONF_SHORT_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3002},
+    (CONF_LONG_RELEASE, CONF_DIM_DOWN): {CONF_EVENT: 3003},
+    (CONF_DOUBLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3004},
+    (CONF_TRIPLE_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 3005},
+    (CONF_LONG_PRESS, CONF_DIM_UP): {CONF_EVENT: 4001},
+    (CONF_SHORT_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4002},
+    (CONF_LONG_RELEASE, CONF_DIM_UP): {CONF_EVENT: 4003},
+    (CONF_DOUBLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4004},
+    (CONF_TRIPLE_PRESS, CONF_DIM_UP): {CONF_EVENT: 4005},
+    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 5001},
+    (CONF_SHORT_RELEASE, CONF_LEFT): {CONF_EVENT: 5002},
+    (CONF_LONG_RELEASE, CONF_LEFT): {CONF_EVENT: 5003},
+    (CONF_DOUBLE_PRESS, CONF_LEFT): {CONF_EVENT: 5004},
+    (CONF_TRIPLE_PRESS, CONF_LEFT): {CONF_EVENT: 5005},
+    (CONF_LONG_PRESS, CONF_RIGHT): {CONF_EVENT:64001},
+    (CONF_SHORT_RELEASE, CONF_RIGHT): {CONF_EVENT: 6002},
+    (CONF_LONG_RELEASE, CONF_RIGHT): {CONF_EVENT: 6003},
+    (CONF_DOUBLE_PRESS, CONF_RIGHT): {CONF_EVENT: 6004},
+    (CONF_TRIPLE_PRESS, CONF_RIGHT): {CONF_EVENT: 6005},
```

Comment:
I would like something like this
```suggestion
    **AQARA_OPPLE_4_BUTTONS,
    (CONF_LONG_PRESS, CONF_DIM_DOWN): {CONF_EVENT: 5001},
    (CONF_SHORT_RELEASE, CONF_LEFT): {CONF_EVENT: 5002},
    (CONF_LONG_RELEASE, CONF_LEFT): {CONF_EVENT: 5003},
    (CONF_DOUBLE_PRESS, CONF_LEFT): {CONF_EVENT: 5004},
    (CONF_TRIPLE_PRESS, CONF_LEFT): {CONF_EVENT: 5005},
    (CONF_LONG_PRESS, CONF_RIGHT): {CONF_EVENT:64001},
    (CONF_SHORT_RELEASE, CONF_RIGHT): {CONF_EVENT: 6002},
    (CONF_LONG_RELEASE, CONF_RIGHT): {CONF_EVENT: 6003},
    (CONF_DOUBLE_PRESS, CONF_RIGHT): {CONF_EVENT: 6004},
    (CONF_TRIPLE_PRESS, CONF_RIGHT): {CONF_EVENT: 6005},
```